libs/sdk-ui-geo/src/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -182,18 +225,30 @@ function renderPushpinCategoryLegend(
     contentRect: ContentRect,
     hasSizeLegend: boolean,
 ): JSX.Element {
-    return <PushpinCategoryLegend {...props} contentRect={contentRect} hasSizeLegend={hasSizeLegend} />;
+    const { containerId, maxRows, name } = props;
+    return (
+        <PushpinCategoryLegend
+            {...props}
+            contentRect={contentRect}
+            hasSizeLegend={hasSizeLegend}
+            containerId={containerId}
```

Comment:
Nitpick: containerId, maxRows and name are already passed with the `...props` above, right?

Reply:
Fixed https://github.com/gooddata/gooddata-ui-sdk/pull/1341/commits/681c4fcc3655d86e0fca2623bae6c458eb23b47b